src/defaults/props.options.js

```diff
@@ -40,6 +40,7 @@ export default {
   selection: false,
   selectionProps: {},
   sorting: true,
+  keepSortOrderOnColumnSwitch: true,
```

Comment:
Settings the default as true would change the behavior of existing tables right?

Reply:
No, it's for being compatible with actual behaviour I set default to true. This means sort order is by default conserved when switching between sorted columns, as it was asked and implemented in this PR: https://github.com/material-table-core/core/commit/a64c4a66681afbb686fc9f84a0ff493c557e48e2

Reply:
Ok perfect, I just wanted to clarify this. Looks good.